app/uk/gov/hmrc/customs/declaration/model/models.scala

```diff
@@ -16,28 +16,34 @@
 
 package uk.gov.hmrc.customs.declaration.model
 
-import uk.gov.hmrc.customs.declaration.model.RequestType.RequestType
-
 case class RequestedVersion(versionNumber: String, configPrefix: Option[String])
 
 case class Eori(value: String) extends AnyVal
 
+case class ClientId(value: String) extends AnyVal
```

Comment:
do we need these models? these are just string. I'd suggest using String, but if we want to use models then better use it in all layers i.e. should be used in ApiSubscriptionKey etc

Reply:
This is quite a big change in the model. Again I will try and address in the next PR